experimental/ner_spancat_compare/configs/ner.cfg

```diff
@@ -0,0 +1,143 @@
+[paths]
+train = null
+dev = null
+vectors = "en_core_web_lg"
+init_tok2vec = null
+
+[system]
+gpu_allocator = null
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["tok2vec","ner"]
```

Comment:
Because this is a comparison project, I'd use the `tok2vec` either as standalone component in both config files, or in neither. I think it'll make it easier for people to compare the two config files :-)

Reply:
Updated here: https://github.com/explosion/projects/pull/103/commits/cd7bdd949c420ace70ebd162a46a345513d028dd